### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
extension-node.com|https://urlscan.io/result/fb6f6728-f4f4-4c74-966a-130599cc8cb7/dom/
cryptoxtmining.com|https://user-images.githubusercontent.com/49607867/165974913-aff102f0-aead-4843-b0ba-9ab65e5fb9e2.png
capital-bittrade.online|https://user-images.githubusercontent.com/49607867/165992957-57e9558e-29e4-4e9b-a7c0-abc5f2a586e2.png
walletvalidatorconnect.com|https://user-images.githubusercontent.com/49607867/165975398-684d532f-8160-4989-9334-f4df7ae8f8f3.png
dexlaunchpads.com|https://urlscan.io/result/e44001ef-f234-4ae5-a747-6992240c3371
decentralizedlauncher.org|https://urlscan.io/result/36bb9dc2-ff8c-4cba-bfc5-b62188dca3b0
wallet-avax.online|https://user-images.githubusercontent.com/49607867/165975008-f9d01a19-2777-448a-a192-c096fa22086e.png
dappsrevalidator.org|https://urlscan.io/result/8732c376-7464-47d8-b097-dfcea6fb10bc/dom/
wallethelp.hb-debt.com|https://urlscan.io/result/e518ac4d-198f-4c28-ba09-6043bba05121/dom/
walletconnectsupport.hb-debt.com|https://urlscan.io/result/f453257f-c5cb-4748-a5a1-11400f14956d/dom/
convalidate.online|https://urlscan.io/result/a95db215-7b5a-4d75-9199-4c791e5d74ee/dom/
393a-168-100-8-171.ngrok.io|https://urlscan.io/result/70464256-8656-4794-99c9-ca8eed1a9cb4/dom/

</body>
</html>

"393a-168-100-8-171.ngrok.io",
"asotplatform.com",
"bancor-dashboard.network",
"capital-bittrade.online",
"convalidate.online",
"cryptoxtmining.com",
"dappsrevalidator.org",
"decentralizedlauncher.org",
"dexlaunchpads.com",
"extension-node.com",
"oandainvest.com",
"othersidemeta-bayc.com",
"wallet-avax.online",
"walletconnectsupport.hb-debt.com",
"wallethelp.hb-debt.com",
"walletvalidatorconnect.com",

### CODE REVIEW

**Comment by dubstard** [comment]
✅Ready for review

File: all.json [hunk 1]
```diff
@@ -14,6 +14,22 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"393a-168-100-8-171.ngrok.io",
```
Comment (jacogr) [inline all.json:17]: Via ngrok? Wow.

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
thanks

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:17]: good old iframe embedded

**Pull request merged**